CLA.md

```diff
@@ -0,0 +1,111 @@
+# Fiduciary License Agreement 2.0
+based on the
+## Individual Contributor Exclusive License Agreement
+(including the Traditional Patent License OPTION)
+
+Thank you for your interest in contributing to maiot GmbH's ZenML ("We" or "Us").
+
+The purpose of this contributor agreement ("Agreement") is to clarify and document the rights granted by contributors to Us. To make this document effective, please follow the instructions at https://zenml.io/cla.
```

Comment:
@benkoller  https://zenml.io/cla is broken for me, with:
```
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>cla</Key>
<RequestId>BC64289244063D73</RequestId>
<HostId>bQcQ/5mua5KyAK3EUCbSPZfwk7HrKzhGKTkL6P7tnFAwO8NpOAqXuhzxax+EZiBLaiR7XQczav0=</HostId>
</Error>
```

Reply:
Just missing a backslash. Added it, will resolve with the next commit.